test_utils.go

```diff
@@ -138,18 +138,18 @@ func createTestPeer(notifier chainntnfs.ChainNotifier,
 	}
 
 	estimator := &lnwallet.StaticFeeEstimator{FeeRate: 50}
-	feePerWeight, err := estimator.EstimateFeePerWeight(1)
+	feePerVSize, err := estimator.EstimateFeePerVSize(1)
 	if err != nil {
 		return nil, nil, nil, nil, err
 	}
-	feePerKw := feePerWeight * 1000
+	feePerKw := feePerVSize.FeePerKWeight()
 
 	// TODO(roasbeef): need to factor in commit fee?
 	aliceCommit := channeldb.ChannelCommitment{
 		CommitHeight:  0,
 		LocalBalance:  lnwire.NewMSatFromSatoshis(channelBal),
 		RemoteBalance: lnwire.NewMSatFromSatoshis(channelBal),
-		FeePerKw:      feePerKw,
+		FeePerKw:      btcutil.Amount(feePerKw),
```

Comment:
Yeah, I wanted to do that, but the compiler throws the error "cyclic imports not allowed". I think it's because `lnwallet` imports `channeldb`, and if I make `FeePerKw` this type, then `channeldb` will import `lnwallet`, causing the cycle. 

Wondering if I should put the types / fee-estimator in a new package? Or if there's a better way to resolve this? :D

Reply:
Yeh the route to resolve this would be to extract all the fee based stuff into a distinct package. Since the import cycle issue would only be encountered in this test file, and it isn't a blocker for this PR, I think we can postpone this movement until later. Generally the `lnwallet` package needs to be broken up, so when we lay out a plan for doing so, we should also move out all the fee estimation related stuff.

Reply:
The same cast is done in `lnwallet/channel.go`, so not only test files are affected. Will add a TODO for this there.